go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/Microsoft/go-winio v0.4.16 // indirect
 	github.com/auth0/go-jwt-middleware/v2 v2.0.0-beta
 	github.com/aws/aws-sdk-go v1.38.41
+	github.com/cenkalti/backoff/v4 v4.0.2
```

Comment:
Do we really need exponential backoff to solve that issue and/or an additional dependency ?

Reply:
I think using a retry backoff strategy is the best way to go considering the indeterministic behaviour in terms of time of AWS. THis library is considering fairly popular and light so I don't think it worths the effort of rewriting the same logic myself.